examples/udfilters.c

```diff
@@ -0,0 +1,171 @@
+/*
+  Copyright (C) 2015  Francesc Alted
+  http://blosc.org
+  License: BSD 3-Clause (see LICENSE.txt)
+
+  Example program demonstrating use of the Blosc filter from C code.
```

Comment:
The comments below does not belong here.  Better remove them.

Reply:
Fixed